web/modules/custom/dbcdk_community/src/Form/FlagsMarkReadForm.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * @file
+ * Class definition for FlaggedContentMarkAllFlagsAsReadForm.
+ */
+
+namespace Drupal\dbcdk_community\Form;
+
+
+use DBCDK\CommunityServices\Api\FlagApi;
+use DBCDK\CommunityServices\ApiException;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\dbcdk_community\Content\FlaggableContent;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Form for marking a list of flags.
+ */
+class FlagsMarkReadForm extends FormBase {
+
+  /**
+   * The flag API to use when marking flags as read.
+   *
+   * @var \DBCDK\CommunityServices\Api\FlagApi
+   */
+  protected $flagApi;
+
+  /**
+   * The logger to use.
+   *
+   * @var LoggerInterface
+   */
+  protected $logger;
+
+  /**
+   * FlagsMarkReadForm constructor.
+   *
+   * @param LoggerInterface $logger
+   *   The logger to use.
+   * @param FlagApi $flag_api
```

Comment:
@kasperg did we agree on using full name spaces, even when the object has been defined with `use` ?
If so, then this should be changes with the LoggerInterface here and in the class property. But I can leave this up to personal preference. It does not bother me